src/tiledb/cloud/utilities/_common.py

```diff
@@ -455,44 +455,42 @@ def find(
     :param max_count: Optional stop point when searching for files
     """
     with tiledb.scope_ctx(config):
-        vfs = tiledb.VFS(config=config, ctx=tiledb.Ctx(config))
-        listing = vfs.ls(uri)
-        current_count = 0
-
-        def list_files(listing):
-            for f in listing:
-                # Avoid infinite recursion
-                if f == uri:
-                    continue
-
-                if vfs.is_dir(f):
-                    yield from list_files(
-                        vfs.ls(f),
-                    )
-                else:
-                    # Skip files that do not match the include pattern or match
-                    # the exclude pattern.
-                    if callable(include):
-                        if not include(f):
-                            continue
-                    else:
-                        if include and not fnmatch(f, include):
-                            continue
-
-                    if callable(exclude):
-                        if exclude(f):
-                            continue
-                    else:
-                        if exclude and fnmatch(f, exclude):
-                            continue
-                    yield f
-
-        for f in list_files(listing):
-            yield f
-
-            current_count += 1
-            if max_count and current_count == max_count:
-                return
+        vfs = tiledb.VFS()
```

Comment:
@gspowley do we need the config and ctx here as in the original code?

Reply:
I believe `with tiledb.scope_ctx(config):` should cover it, but we can add config and ctx back to minimize changes.